src/App.js

```diff
@@ -1,16 +1,42 @@
 import React, { Component } from 'react'
 import { Instructions } from './components'
+import { Counter } from './components'
 import './styles/app.css'
 
 class App extends Component {
+  //  var mylist = ["Ayy", "whats", "good"]
+  //  const mapper = list.map(x : x + "\n")
   render() {
+    let mylist = ['Ayy', 'whats', 'good']
+    mylist = mylist.filter(x => x.length > 2)
+    mylist = mylist.map((x, index) => (index % 2 === 1 ? x.toUpperCase() : x))
```

Comment:
you can chain the `filter` and `map`s and assign the result of all the transformations to a variable:
```javascript
const mylist = items.filter(item => item.length > 2)
  .map(item => ...)
  .filter(item => ...)
  .map(blah => blah blah)